account_move_line_tax_editable/views/account_move.xml

```diff
@@ -13,11 +13,27 @@
             >
                 <field name="is_tax_editable" invisible="1" />
                 <field
-                    name="tax_line_id"
-                    attrs="{'readonly': [('is_tax_editable', '=', False)]}"
-                    optional="hide"
+                    name="force_tax_line_id"
+                    attrs="{'column_invisible': [('parent.state', '!=', 'draft')]}"
```

Comment:
use `is_tax_editable` for the condition.

Reply:
I couldn't use it here, because `is_tax_editable` is an account.move.line field, but the view is account.move. Column_invisible is an attribute of the parent view that cannot be modified using line filelds, cause it could have a different value for each record.